docker-build-env/entrypoint.sh

```diff
@@ -0,0 +1,8 @@
+#!/bin/sh
+
+# Initialize softhsm and create a key
```

Comment:
This seems like test logic to me. I don't think the user would expect a key to be created when the Docker image is created.

Reply:
Done.